### PR TITLE
Support more datatypes in InfluxValueField 

### PR DESCRIPTION
**Pull request by bqstony**
In my case i write data from a PLC to influxdb. This 32bit controller result the data in smaler size as regular used in .net. So i have extended the conversion form source types.

I also have added a interface for `InfluxValueField`, so it is a lot more confortable to implement own behavior based on special datatypes.

Give me some feedback if needed. I can also write some test if you like 😎

### CODE REVIEW

File: src/DataStructures/InfluxDatapoint.cs [hunk 1]
```diff
@@ -142,7 +148,11 @@ public string ConvertToInfluxLineProtocol()
             else if (tType == typeof(double))
                 ////double has to have a . as decimal seperator for Influx
                 fields = String.Join(",", Fields.Select(v => new StringBuilder().AppendFormat("{0}={1}", v.Key.EscapeChars(comma: true, equalSign: true, space: true), String.Format(new CultureInfo("en-US"), "{0}", v.Value))));
-            else if (tType == typeof(InfluxValueField))
+#if NETSTANDARD1_6
```
Comment (bqstony) [inline src/DataStructures/InfluxDatapoint.cs]: If we can update to .net standard 2.0 is is not necessary anymore!

Comment (mvadu) [reply src/DataStructures/InfluxDatapoint.cs]: i think we need to update it to .net standard 2, its been a while. I still support .Net 4.5 on the non core side :)

**Comment by mvadu** [comment]
Please update the minor version in both .Net legacy side and Core side. Also please update change log with this minor version details.

File: src/DataStructures/InfluxDatapoint.cs (hunk 1 above)
Comment (bqstony) [reply src/DataStructures/InfluxDatapoint.cs]: With .Net Standard 2.0 the project seperation is not required enymore!?! The library can be used easy with .net 4.6.1 onwoard.

Comment (bqstony) [reply src/DataStructures/InfluxDatapoint.cs]: I would suggest to create a last nuget for 
AdysTech.InfluxDB.Client.Net and add a dependency to the AdysTech.InfluxDB.Client.Net.Core
Projekt. In this way only one nuget is relevant. What you think?

**Comment by mvadu** [comment]
Thank you for this great contribution @bqstony. I was out last few days. Merging it today. I will push the nuget package little later.

**Pull request merged**